packages/mobile/src/identity/contactMapping.ts

```diff
@@ -301,7 +305,7 @@ export function getAddressesFromLookupResult(
     verifiedAddresses.push(address.toLowerCase())
   }
 
-  return verifiedAddresses.length ? verifiedAddresses : null
+  return verifiedAddresses
```

Comment:
Can we rename this to `verifiedAccountAddresses`? I think it would be helpful to be as clear as possible about the addresses we are getting given the confusion over when we are dealing `accountAddress` vs `walletAddress`